template/src/components/app.js

```diff
@@ -1,6 +1,10 @@
 import { h, Component } from 'preact';
 import { Router } from 'preact-router';
 
+if (module.hot) {
+	require('preact/debug');
+}
+
```

Comment:
I'd prefer moving this to the end of import statements, just looks a bit odd in the middle 🙈

Reply:
✅ cf01986

I only put it there because I usually do all npm imports first, then local imports, but I'm absolutely fine with putting it at the bottom